planner/cascades/testdata/integration_suite_out.json

```diff
@@ -531,12 +530,13 @@
       {
         "SQL": "select * from (select a+1 as c, a+b as d from t) as t1 order by c+d limit 10",
         "Plan": [
-          "Projection_22 10.00 root plus(test.t.a, 1)->Column#3, plus(test.t.a, test.t.b)->Column#4",
-          "└─TopN_23 10.00 root Column#5:asc, offset:0, count:10",
-          "  └─Projection_25 10.00 root test.t.a, test.t.b, plus(plus(test.t.a, 1), plus(test.t.a, test.t.b))->Column#5",
-          "    └─TableReader_26 10.00 root data:TopN_27",
-          "      └─TopN_27 10.00 cop[tikv] plus(plus(test.t.a, 1), plus(test.t.a, test.t.b)):asc, offset:0, count:10",
-          "        └─TableFullScan_28 10000.00 cop[tikv] table:t, keep order:false, stats:pseudo"
+          "Projection_22 4.00 root plus(test.t.a, 1)->Column#3, plus(test.t.a, test.t.b)->Column#4",
+          "└─Limit_24 4.00 root offset:0, count:10",
+          "  └─Sort_30 4.00 root Column#5:asc",
+          "    └─Projection_25 4.00 root test.t.a, test.t.b, plus(plus(test.t.a, 1), plus(test.t.a, test.t.b))->Column#5",
```

Comment:
`TopN` uses a heap whose size (10) is larger than the row count of its child (4). So the cost of `TopN` > the cost of `Limit + Sort`. 
I'm not sure if we can accept such change. One solution is to use `property.Enfored` again in cascades planner.